vars/deployPlannerSnapshot.groovy

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/groovy
+
+import java.util.regex.Pattern
+
+def call(body) {
+    // evaluate the body block, and collect configuration into the object
+    def config = [:]
+    body.resolveStrategy = Closure.DELEGATE_FIRST
+    body.delegate = config
+    body()
+
+    def yaml
+    def openShiftProject = config.openShiftProject
```

Comment:
how about `def openshiftProject = config.openshiftProject + '-' + utils.getRepoName()`
so that this variable isn't overwritten in line 24?

Reply:
yes! done.